tools/bowtie2.wdl

```diff
@@ -0,0 +1,410 @@
+        nofw: "Do not align to the forward (original) strand of the reference"
+        norc: "Do not align to the reverse-complement strand of the reference"
+        no_1mm_upfront: {
+            description: "Do not allow 1 mismatch alignments before attempting to scan for the optimal seeded alignments",
+            help: "By default, Bowtie 2 will attempt to find either an exact or a 1-mismatch end-to-end alignment for the read before trying the multiseed heuristic. Such alignments can be found very quickly, and many short read alignments have exact or near-exact end-to-end alignments. However, this can lead to unexpected alignments when the user also sets options governing the multiseed heuristic, like `seed_subtring` and `seed_mismatch`. For instance, if the user specifies `seed_mismatch` 0 and `seed_substring` equal to the length of the read, the user will be surprised to find 1-mismatch alignments reported. This option prevents Bowtie 2 from searching for 1-mismatch end-to-end alignments before using the multiseed heuristic, which leads to the expected behavior when combined with options such as `seed_substring` and `seed_mismatch`. This comes at the expense of speed.",
+        }
+        end_to_end: {
+            description: "If true, entire read must align; no clipping. Else, local alignment; ends might be soft clipped",
+            help: "If true, Bowtie 2 requires that the entire read align from one end to the other, without any trimming (or soft clipping) of characters from either end. The match bonus always equals 0 in this mode, so all alignment scores are less than or equal to 0, and the greatest possible alignment score is 0. If false, Bowtie 2 does not require that the entire read align from one end to the other. Rather, some characters may be omitted (soft clipped) from the ends in order to achieve the greatest possible alignment score. The match bonus is used in this mode, and the best possible alignment score is equal to the match bonus times the length of the read.",
+        }
+        match_bonus: {
+            description: "bonus for match (0 for end-to-end or 2 for local)",
+            bowtie2_option: "ma",
+        }
+        mismatch_penalty: {
+            description: "max penalty for mismatch; lower quality = lower penalty",
+            help: "Sets the maximum (MX) and minimum (MN) mismatch penalties, both integers. A number less than or equal to MX and greater than or equal to MN is subtracted from the alignment score for each position where a read character aligns to a reference character, the characters do not match, and neither is an N. If `ignore_quals` is specified, the number subtracted quals MX. Otherwise, the number subtracted is MN + floor( (MX-MN)(MIN(Q, 40.0)/40.0) ) where Q is the Phred quality value.",
+            bowtie2_option: "mp",
+        }
+        non_actg_penalty: "penalty for non-A/C/G/Ts in read/reference"
```

Comment:
Can this be expanded a little? I think I get it, but some more naive users might not.